application/controllers/Hook.php

```diff
@@ -14,6 +14,8 @@ public function add()
 
         //is there a GCP URL in environment variable ?
         $this->GCPURL = $this->config->item('GCP_URL').'reports/';
+        //TODO
+        $this->GCPURL = '/home/sgarny/Bureau/';
```

Comment:
@SimonGrn Normal ?

Reply:
Oops :sweat_smile: no it's not, I need to work on it tho, there's still a lot of problems 
I'll fix it right now